charts/index.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+entries:
+  secrets-store-csi-driver:
+  - apiVersion: v1
+    appVersion: 0.0.9
+    created: "2020-04-27T23:49:22.185487+01:00"
+    description: A Helm chart to install the SecretsStore CSI Driver inside a Kubernetes
+      cluster.
+    digest: ed46bde0f57574ba84fade57f6c2222899fbe8917a051912990576e3510d974e
+    icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
+    kubeVersion: '>=1.15.0-0'
+    maintainers:
+    - email: ritazh@microsoft.com
+      name: Rita Zhang
+    name: secrets-store-csi-driver
+    sources:
+    - https://github.com/kubernetes-sigs/secrets-store-csi-driver
+    urls:
+    - https://raw.githubusercontent.com/contino/secrets-store-csi-driver/master/charts/secrets-store-csi-driver-0.0.9.tgz
```

Comment:
The url should be - https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts/secrets-store-csi-driver-0.0.9.tgz

```bash
helm repo index . --url https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts/
```

Reply:
Apologies this made it in from a separate upstream.